Sources/MapboxMaps/Location/Pucks/LocationPuckManager.swift

```diff
@@ -27,6 +19,13 @@ public enum PuckType: Equatable {
     case puck3D(Puck3DConfiguration)
 }
 
+// MARK: PuckBearingSource
+/// This enum controls how the puck is oriented
+public enum PuckBearingSource: Equatable {
+    case heading
+    case course
```

Comment:
Could you add some documentation here that goes into more detail about these individual cases?